apps/src/applab/applab.js

```diff
@@ -645,7 +631,7 @@ Applab.init = function (config) {
     // should never be present on such levels, however some levels do
     // have levelHtml stored due to a previous bug. HTML set by levelbuilder
     // is stored in startHtml, not levelHtml.
-    if (Applab.reduxStore.getState().level.isDesignModeHidden) {
+    if (studioApp.reduxStore.getState().level.isDesignModeHidden) {
```

Comment:
Somehow this feels even uglier with the store on studioApp... but that might be a good thing, as it will discourage us from doing this too much?